app/overrides/spree/shared/_head/add_bolt_embed_script.html.erb.deface

```diff
@@ -3,3 +3,4 @@
 <meta name="bolt-publishable-key" content="<%= SolidusBolt::BoltConfiguration.fetch.publishable_key %>" />
 <meta name="bolt-domain" content="<%= SolidusBolt::BoltConfiguration.fetch.environment_url%>" />
 <script id="bolt-embedded" type="text/javascript" src="<%= SolidusBolt::BoltConfiguration.fetch.embed_js %>"></script>
+<script id="bolt-account" type="text/javascript" src="<%= SolidusBolt::BoltConfiguration.fetch.account_js %>" data-publishable-key="<%= SolidusBolt::BoltConfiguration.fetch.publishable_key %>"></script>
```

Comment:
Why you didn't add the async attributes?

Reply:
I was debugging an issue and had removed that attribute.
Added it back now here [edccbba](https://github.com/nebulab/solidus_bolt/commit/edccbba714837a46c4f9da221de6c4dc18a37275).

Bolt login/register modal working fine.